consts.js

```diff
@@ -25,13 +25,14 @@ var thirteenStrings = [
     "PT", // PT is Brazilian political party represented by the number 13
     "Washington Luís", // Brazil's thirteenth president
     "Millard Fillmore", // Thirteenth President of the United States
+    "Louis XIII" // Thirteenth king of France
```

Comment:
Trajan was a pretty awesome dude, but why Louis? What good things did he do?

Reply:
He was gay and made men wearing wigs fashionable?

Reply:
good point.